server.py

```diff
@@ -126,11 +141,12 @@ def update_sign():
         })
     except Exception as e:
         print(e, flush=True)
-        sign_message = None
+        sign_message = SignMessage(initial)
```

Comment:
i think we should keep this line as it was, 

```suggestion
        sign_message = None
```

its in the javascript that we need to handle the response from /api/health-check being empty. if it is, show some simple text in the html like "sign is off!"